### PR TITLE
Finished IL static load

### PR DESCRIPTION
**Pull request by nrpriore**

### CODE REVIEW

File: Assets/Scripts/GameData/GameData.cs [hunk 1]
```diff
@@ -90,11 +138,31 @@ public static class GameData {
...
+	// Called to retrieve static action data
+	public static ActionData Actions(string nameKey) {
+		return actions.Find(x => x.name == nameKey);
+	}
+
+	// Called to retrieve static perk data
+	public static PerkData Perks(string nameKey) {
+		return perks.Find(x => x.name == nameKey);
+	}
+
```
Comment (akkowal2) [inline Assets/Scripts/GameData/GameData.cs:170]: Are these getter functions?  If so maybe consider renaming these like getPerk, getAction, etc.  Actions(param) seems more like a constructor for a class than it does a getter function.

File: Assets/Scripts/GameData/StatData.cs [hunk 2]
```diff
@@ -5,11 +5,16 @@ public class StatData {
 
 	public string name;
 	public string description;
+	public float value;
```
Comment (akkowal2) [inline Assets/Scripts/GameData/StatData.cs:9]: Is value a keyword in C#?  It looks like all three of these words are showing up as pink.

Comment (nrpriore) [reply Assets/Scripts/GameData/StatData.cs:9]: It is, though not in this case. can change it though if we feel it necessary

File: Assets/Scripts/GameData/GameData.cs (hunk 1 above)
Comment (nrpriore) [reply Assets/Scripts/GameData/GameData.cs:170]: These are indeed, and I agree - will update

File: Assets/Scripts/GameData/StatData.cs (hunk 2 above)
Comment (akkowal2) [reply Assets/Scripts/GameData/StatData.cs:9]: I looked it up.  Looks like it is more of a contextual keyword used in set blocks.  Seeing as that is the case, I don't feel strongly either way.  Just wanted to make sure we weren't overloading a language keyword or doing something funky.

File: Assets/Scripts/GameData/UnitData.cs [hunk 3]
```diff
@@ -7,10 +7,21 @@ public class UnitData {
...
+
+		stats = new List<StatData>();
+		foreach(KeyValuePair<string, object> stat in (Dictionary<string, object>)unitData["Stats"]) {
+			stats.Add(new StatData(stat, true));
+		}
+	}
+
+	public StatData Stats(string nameKey) {
+		return stats.Find(x => x.name == nameKey);
 	}
```
Comment (akkowal2) [inline Assets/Scripts/GameData/UnitData.cs:25]: Missed a rename for getters?

File: Assets/Scripts/GameData/LeaderData.cs [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+		abilities = new List<AbilityData>();
+		foreach(string ability in Json.ToList(leaderData["Abilities"].ToString())) {
+			abilities.Add(new AbilityData(ability));
+		}
+	}
+
+	public StatData Stats(string nameKey) {
+		return stats.Find(x => x.name == nameKey);
+	}
+
```
Comment (akkowal2) [inline Assets/Scripts/GameData/LeaderData.cs:32]: Getter rename?

**Review by akkowal2 — APPROVED** [review]

**Pull request merged**